pkg/scheduler/predicates/ha.go

```diff
@@ -448,3 +456,16 @@ func getTopologyFromNode(topologyKey string, nodeName string, nodes []apiv1.Node
 	}
 	return ""
 }
+
+func isPodDesired(tc *v1alpha1.TidbCluster, component, podName string) bool {
```

Comment:
@cvvz Could you please add some UT cases for this function?

Reply:
@DanielZhangQD sure.